File: src/Models/Entities/OrderHeader.cs
using System;
namespace Models.Entities
{
    public class OrderHeader
    {
        public int OrderId { get; set; }
        public string OrderNo { get; set; }
        public int UserId  { get; set; }
        public DateTime OrderDate { get; set; }
        public int Status  { get; set; }      
        public DateTime OrderTransactionTime { get; set; }
        public int DeliveryTimeSlotId { get; set; }
        public bool IsDelivered { get; set; }
         
    }
}

File: src/ClientApp/src/app/models/OrderHeader.ts
import { OrderDetail } from "./OrderDetail";

export class OrderHeader {
    orderHeaderId: number;
    orderNo: string;
    userId:number;
    orderDate:string;
    totalAmount:number
    lineItemCount: number;
    expectedDeliveryDate: string;
    status: number;
    orderDetail: OrderDetail[]
  }